src/main/fc/rc_controls.c

```diff
@@ -118,11 +119,20 @@ throttleStatus_e calculateThrottleStatus(void)
     return THROTTLE_HIGH;
 }
 
+#define ARM_DELAY_TIME 500000 //arm,disarm delay usec
+#define STICK_DELAY_RATIO 10  //stick command delay usec = ARM_DELAY_TIME/STICK_DELAY_RATIO
 void processRcStickPositions(throttleStatus_e throttleStatus)
 {
-    static uint8_t rcDelayCommand;      // this indicates the number of time (multiple of RC measurement at 50Hz) the sticks must be maintained to run or switch off motors
-    static uint8_t rcSticks;            // this hold sticks position for command combos
-    static uint8_t rcDisarmTicks;       // this is an extra guard for disarming through switch to prevent that one frame can disarm it
+    // RC refresh rate converted to number of time the sticks must be maintained to arm,disarm
+    uint8_t rcDelayTicks = constrain(ARM_DELAY_TIME / getTaskDeltaTime(TASK_RX), 20, 50);
```

Comment:
Why are we constraining here? The way `rcDelayTicks` is used is counter-intuitive anyway, we should probably just use the expression where it is checked. This will also allow us to get rid of `STICK_DELAY_RATIO` and use `STICK_DELAY_TIME_US` that is independent of `ARM_DELAY_TIME`.

Reply:
Seems like this value should be a part of an appropriate `init` function, there's no need to calculate it on each call to `processRcStickPositions`.

Reply:
I don't really understand the nature of constraining here either as it seems to break the relation between `rcDelayTicks` and real time it takes to register a stick command should the quotient fall outside of [20, 50]